packages/hooks/src/__tests__/useFiatAccountList.spec.tsx

```diff
@@ -1,78 +1,133 @@
+                        AUD: { type: 'fiat', is_crypto: false },
+                        BTC: { type: 'crypto', is_crypto: true },
+                        ETH: { type: 'crypto', is_crypto: true },
+                        UST: { type: 'crypto', is_crypto: true },
+                        USD: { type: 'fiat', is_crypto: false },
+                        LTC: { type: 'crypto', is_crypto: true },
+                    },
+                },
             },
-        });
+            isLoading: false,
+            isSuccess: true,
+        } as unknown as ReturnType<typeof mockUseFetch>);
 
-        const wrapper = ({ children }: { children: JSX.Element }) => (
-            <StoreProvider store={mock}>{children}</StoreProvider>
-        );
+        const wrapper = ({ children }: { children: JSX.Element }) => <APIProvider>{children}</APIProvider>;
 
         const { result } = renderHook(() => useFiatAccountList(), { wrapper });
 
         expect(result.current).toStrictEqual([]);
     });
 
     test('should return list of fiat accounts if client any fiat account', async () => {
-        const mock = mockStore({
-            client: {
-                account_list: [
-                    { title: 'BTC', is_virtual: false, loginid: 'CR123' },
-                    { title: 'USD', is_virtual: false, loginid: 'CR123' },
-                ],
-                is_crypto: (currency: string) => currency === 'BTC',
+        mockUseFetch.mockReturnValue({
+            data: {
+                authorize: {
+                    account_list: [
+                        { currency: 'BTC', is_virtual: 0, loginid: 'CR123' },
+                        { currency: 'USD', is_virtual: 0, loginid: 'CR456' },
+                    ],
+                    loginid: 'CR123',
+                },
+                website_status: {
+                    currencies_config: {
+                        AUD: { type: 'fiat', is_crypto: false },
+                        BTC: { type: 'crypto', is_crypto: true },
+                        ETH: { type: 'crypto', is_crypto: true },
+                        UST: { type: 'crypto', is_crypto: true },
+                        USD: { type: 'fiat', is_crypto: false },
+                        LTC: { type: 'crypto', is_crypto: true },
+                    },
+                },
             },
-        });
+            isLoading: false,
+            isSuccess: true,
+        } as unknown as ReturnType<typeof mockUseFetch>);
 
-        const wrapper = ({ children }: { children: JSX.Element }) => (
-            <StoreProvider store={mock}>{children}</StoreProvider>
-        );
+        const wrapper = ({ children }: { children: JSX.Element }) => <APIProvider>{children}</APIProvider>;
 
         const { result } = renderHook(() => useFiatAccountList(), { wrapper });
 
-        expect(result.current).toStrictEqual([{ title: 'USD', is_virtual: false, loginid: 'CR123' }]);
+        expect(result.current).toHaveLength(1);
```

Comment:
@sergei-deriv why did you replace this check with just checking of length?

Reply:
@mahdiyeh-deriv It's because I added a lot of properties to accounts list and was too hard to write it all. I'll rewrite it with `toMatchObject` method instead of `toStrictEqual`. Thanks for your attention